src/redux/book/book.js

```diff
@@ -0,0 +1,42 @@
+const ADD_A_BOOK = 'bookStore/book/ADD_A_BOOK';
+const REMOVE_A_BOOK = 'bookStore/book/REMOVE_A_BOOK';
+const LIST_BOOKS = 'bookStore/book/LIST_BOOKS';
```

Comment:
- [ ] [OPTIONAL] Kindly remove the `LIST_BOOKS` action creator as it's redundant. It's the same as the default case.